00-packer.json

```diff
@@ -1,7 +1,8 @@
 {
   "variables": {
     "aws_access_key": "{{env `AWS_ACCESS_KEY`}}",
-    "aws_secret_key": "{{env `AWS_SECRET_KEY`}}"
+    "aws_secret_key": "{{env `AWS_SECRET_KEY`}}",
+    "tag": "{{env `CIRCLE_BRANCH`}}"
```

Comment:
Ahh, probably.

I think I was operating off the assumption that someone would run ```packer inspect``` which prints out the configurable variables. I'll add that before merging.